metacat-main/src/main/java/com/netflix/metacat/main/services/GetDatabaseParameters.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.main.services;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Getdatabase Parameters.
+ * @author zhenl
+ * @since 1.2.0
+ */
+@Data
```

Comment:
Take a look at Lombok's [Value](https://projectlombok.org/features/Value)

Reply:
Will change to `@Value`, which makes senses here.